cluster/stone-prd-rh01/managed/rhtap-releng-tenant/rb-ecp.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: tenant-ecp
+  namespace: rhtap-releng-tenant
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: release-pipeline-resource-role
```

Comment:
Where's this role defined?

Reply:
> Where's this role defined?

https://github.com/redhat-appstudio/infra-deployments/blob/53dabaeac9cba2946048579bdf5ddc2ac21d0db7/components/release/base/release-pipeline-resources-clusterrole.yaml#L5

Reply:
Oh - I think I chose the wrong role. This would over-grant rights to the `appstudio-pipeline` SA.

Reply:
Ok, fixed in 9105396.